content/en/user-guide/polaris-catalog/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Polaris Catalog"
+linkTitle: "Polaris Catalog"
+weight: 18
+description: Get started with Polaris Catalog in LocalStack for Snowflake
+---
+
+{{< preview-notice >}}
+
+## Introduction
+
+Polaris Catalog is a unified data catalog that provides a single view of all your data assets across Snowflake and external sources. It enables you to discover, understand, and govern your data assets, making it easier to find and use the right data for your analytics and machine learning projects.
+
+The Snowflake emulator supports creating Iceberg tables with Polaris catalog. Currently, [`CREATE CATALOG INTEGRATION`](https://docs.snowflake.com/en/sql-reference/sql/create-catalog-integration-open-catalog) is supported by LocalStack.
+
+## Getting started
+
+This guide is designed for users new to Iceberg tables with Polaris catalog and assumes basic knowledge of SQL and Snowflake. Start your Snowflake emulator and connect to it using an SQL client in order to execute the queries further below.
+
+In this guide, you will create an Iceberg table, display the data in the Iceberg table, and drop the Iceberg table.
+
+### Create an Iceberg table
+
+You can create an Iceberg table using the `CREATE ICEBERG TABLE` statement. In this example, you can create an external volume called `v_demo`:
```

Comment:
```suggestion
You can create an Iceberg table using the `CREATE ICEBERG TABLE` statement. In this example, you can create an external volume called `v_demo` (note that this volume points to a local S3 bucket in the LocalStack AWS emulator):
```

Reply:
On a related note - is it required to create the `testbucket` S3 bucket ahead of time? (haven't tried it out myself, but just to confirm..)